### PR TITLE
fix: implement changes keeping metatags

### PR DESCRIPTION
**Pull request by gabriel-azion**

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/architectures/bot-management/bot-management-architecture.mdx [hunk 1]
```diff
@@ -1,9 +1,14 @@
 ---
 title: Protect Your Edge Applications with a Bot Management Solution
-description: Use Azion Bot Manager to protect edge applications from bots, preventing malicious attacks.
-meta_tags: architecture, secure, bot, bot management, security, firewall
-namespace: docs_arch_bot_management
+description: Discover Azion's Bot Manager leveraging machine learning for advanced
+  bot detection, ensuring robust security with real-time analytics on the edge platform.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/bot-management/bot-management-architecture.mdx:4]:
```suggestion
description: Discover Azion Bot Manager, which leverages machine learning for advanced bot detection, ensuring robust security with real-time analytics on the edge platform.
```

File: src/content/docs/en/pages/architectures/edge-application/content-delivery.mdx [hunk 2]
```diff
@@ -1,9 +1,14 @@
 ---
 title: Boost Content Delivery Speed and Reliability at the Edge
-description: Use edge-native content to replace legacy CDNs and leverage the speed and reliability of the edge network while reducing costs.
-meta_tags: edge, applications, web applications, api, content delivery, cdn
-namespace: docs_arch_content_delivery
+description: Explore how Azion Edge Platform enhances content delivery with optimized
+  cache policies, dynamic content handling, and improved user experiences.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-application/content-delivery.mdx:4]: Sugestão: Explore how Azion Edge Platform enhances content delivery with optimized cache policies and dynamic content handling, improving user experiences.

File: src/content/docs/en/pages/architectures/edge-application/image-processing.mdx [hunk 3]
```diff
@@ -1,9 +1,15 @@
 ---
 title: Optimize and process images with Azion Edge Platform
-description: Use Azion Image Processor to automate resource management, reduce load times and storage costs, and leverage the speed and reliability of the edge network.
-meta_tags: edge, applications, web applications, api, content delivery, cdn, image processing
-namespace: docs_arch_image_processing
+description: Explore Azion Image Processor for efficient Edge Computing solutions.
+  Automate resource management, optimize images, and streamline content delivery with
+  advanced cache policies.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-application/image-processing.mdx:5]:
```suggestion
description: Explore Azion Image Processor to automate resource management, optimize images, and streamline content delivery with advanced cache policies.
```

File: src/content/docs/en/pages/architectures/edge-firewall/waap-architecture.mdx [hunk 4]
```diff
@@ -1,9 +1,14 @@
 ---
 title: Enhance Cybersecurity with Azion's Advanced WAAP Solution
-description: Use Azion WAAP solution to secure applications and APIs against cyber threats from OWASP Top 10, sophisticated zero-day attacks, and automated bot activities.
-meta_tags: architecture, secure, firewall, waap, web application and api protection, waf
-namespace: docs_arch_waap
+description: Explore Azion's WAAP for advanced protection from cyber threats including
+  OWASP Top 10 & zero-day attacks using features like WAF & DDoS Protection.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-firewall/waap-architecture.mdx:4]:
```suggestion
description: Explore Azion's WAAP for advanced protection from cyber threats including
  OWASP Top 10 and zero-day attacks using products like WAF and DDoS Protection.
```

File: src/content/docs/en/pages/architectures/live-streaming/live-streaming-delivery-hls.mdx [hunk 5]
```diff
@@ -1,9 +1,15 @@
 ---
 title: Optimize the distribution of video content with live streaming delivery
-description: This architecture leverages advanced features like real-time transcoding, adaptive bitrate streaming, and intelligent caching, enabling seamless experiences for viewers.
-meta_tags: architecture, content delivery, hls, edge computing
-namespace: docs_arch_live_streaming_delivery
+description: Explore how Edge Computing optimizes live streaming with HLS, real-time
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/live-streaming/live-streaming-delivery-hls.mdx:3]:
```suggestion
description: Explore how edge computing optimizes live streaming with HLS, real-time
```

File: src/content/docs/en/pages/architectures/security-modernization/security-modernization-with-edge-computing.mdx [hunk 6]
```diff
@@ -1,9 +1,15 @@
 ---
 title: Accelerate security modernization with edge computing
-description: Edge computing capabilities can help you safeguard your applications and their content, benefiting you from modern logic and observability products.
-meta_tags: architecture, secure, edge firewall
-namespace: docs_arch_security_modernization_with_edge
+description: Explore how Edge Computing enhances security with Edge Firewall, DDoS
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/security-modernization/security-modernization-with-edge-computing.mdx:3]:
```suggestion
description: Explore how edge computing enhances security with Edge Firewall, DDoS
```

File: src/content/docs/en/pages/build-journey/advanced-configurations/multiple-origins.mdx [hunk 7]
```diff
@@ -1,10 +1,15 @@
 ---
 title: How to configure multiple origins with load balancing algorithms
-description: Prevent server overload and traffic congestion across origins, data centers, or cloud providers through the load balancing algorithms provided by Azion.
-meta_tags: build
-namespace: docs_guides_build_advanced_multi_origins
+description: Explore how to configure Load Balancer for Edge Applications on Azion,
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/advanced-configurations/multiple-origins.mdx:3]:
```suggestion
description: Explore how to configure Load Balancer for Edge Application on Azion,
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/build-journey/advanced-configurations/multiple-origins.mdx:3]: OR `edge applications`

File: src/content/docs/en/pages/build-journey/build-with-edge-app/build-with-edge-apps.mdx [hunk 8]
```diff
@@ -1,11 +1,14 @@
 ---
 title: Build an application
-description: Boost the performance of your applications with Azion Edge Application.
-meta_tags: build, edge application, edge network, edge computing
+description: Explore how to deploy and configure your Edge Applications with Azion's
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/build-with-edge-app/build-with-edge-apps.mdx:3]:
```suggestion
description: Explore how to deploy and configure your edge applications with Azion's
```

File: src/content/docs/en/pages/build-journey/develop-with-azion/frameworks/framework-overview.mdx [hunk 9]
```diff
@@ -1,10 +1,14 @@
 ---
 title: Overview
-description: Frameworks.
-meta_tags: js, jamstack
-namespace: docs_framework_overview
+description: Explore the revolution of Jamstack architecture with Edge Computing,
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/develop-with-azion/frameworks/framework-overview.mdx:3]:
```suggestion
description: Explore the revolution of Jamstack architecture with edge computing,
```

File: src/content/docs/en/pages/build-journey/develop-with-azion/frameworks/framework-overview.mdx [hunk 10]
```diff
@@ -1,10 +1,14 @@
...
+  using tools like Vulcan and Azion CLI for efficient web app development on the Azion
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/develop-with-azion/frameworks/framework-overview.mdx:4]:
```suggestion
  using tools like Vulcan and Azion CLI for efficient web application development on the Azion's
```

File: src/content/docs/en/pages/build-journey/develop-with-azion/lang-js/lang-javascript.mdx [hunk 11]
```diff
@@ -1,11 +1,14 @@
 ---
 title: How to build edge functions
-description: Take the steps to become an edge developer by developing edge functions.
-meta_tags: serverless functions, edge, javascript
-namespace: documentation_dev_with_azion_lang_js
-meta_tag_robots_no_index: false
+description: Explore how to develop and debug serverless Edge Functions using JavaScript
+  on the Azion Edge Runtime, enhancing your Edge Applications.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/develop-with-azion/lang-js/lang-javascript.mdx:4]:
```suggestion
  on the Azion Edge Runtime, enhancing your edge applications.
```

File: src/content/docs/en/pages/build-journey/develop-with-azion/lang-wasm/lang-wasm.mdx [hunk 12]
```diff
@@ -1,10 +1,13 @@
 ---
 title: How to build with WebAssembly
-description: Learn about the process required to create an edge function that uses a WebAssembly originated function.
-meta_tags: javascript, edge computing, wasm, webassembly
-namespace: documentation_dev_with_azion_lang_wasm
+description: Explore how to create Edge Functions using JavaScript and WebAssembly
+  on Azion, including guide on wasmCode implementation for Edge Computing.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/develop-with-azion/lang-wasm/lang-wasm.mdx:4]:
```suggestion
description: Explore how to create edge functions using JavaScript and WebAssembly
  on Azion, including a guide on wasmCode implementation for edge computing.
```

File: src/content/docs/en/pages/build-journey/develop-with-azion/toolslocal-dev/local-dev.mdx [hunk 13]
```diff
@@ -1,10 +1,13 @@
 ---
 title: Local development
-description: Local dev
-meta_tags: edge, javascript
-namespace: documentation_dev_with_azion_local_dev
+description: Explore how Azion CLI enhances local development for Edge Applications
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/develop-with-azion/toolslocal-dev/local-dev.mdx:3]:
```suggestion
description: Explore how Azion CLI enhances local development for edge applications
```

File: src/content/docs/en/pages/build-journey/edit-edge-app/edit-device-groups/device-groups.mdx [hunk 14]
```diff
@@ -1,10 +1,14 @@
 ---
 title: How to create device groups
-description: Group end-users using Device Groups based on their device, operating system, or browser details accessed via the User-Agent HTTP request header.
-meta_tags: build, device, user agent, browser, detect
-namespace: docs_guides_build_create_device_groups
+description: Learn how to create and manage Device Groups in Azion Console to tailor
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/edit-edge-app/edit-device-groups/device-groups.mdx:3]:
```suggestion
description: Learn how to create and manage Device Groups on Azion to tailor
```

File: src/content/docs/en/pages/build-journey/troubleshooting/debug-apps.mdx [hunk 15]
```diff
@@ -1,10 +1,14 @@
 ---
 title: Debug applications
-description: Follow the debugging process to check and identify error in your applications.
-meta_tags: observe, debug, debugging, applications, functions, build
-namespace: docs_build_debug_apps
+description: Learn how to debug Edge applications and functions using rules engine,
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/troubleshooting/debug-apps.mdx:3]:
```suggestion
description: Learn how to debug edge applications and functions using rules engine,
```

File: src/content/docs/en/pages/build-journey/troubleshooting/understanding-metrics.mdx [hunk 16]
```diff
@@ -1,10 +1,14 @@
 ---
 title: Understand metrics
-description: You can continuously monitor your applications through Azion Observe products.
-meta_tags: observe, metrics, applications, build
-namespace: docs_build_understanding_metrics
+description: Explore how Azion Observe enhances monitoring of Edge Applications and
+  Functions with Real-Time Metrics, Events, and customized API data queries.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/troubleshooting/understanding-metrics.mdx:4]:
```suggestion
description: Explore how Azion Observe enhances the monitoring of edge applications and
  functions with Real-Time Metrics, Events, and customized API data queries.
```

File: src/content/docs/en/pages/changelog/previous-years/previous-years-changelog.mdx [hunk 17]
```diff
@@ -1,9 +1,13 @@
 ---
 title: Changelog previous years
-description: Follow all changes and updates of Azion products and services.
-meta_tags: Changes in products and services, Changes in products and services by Azion, Azion services update
-namespace: documentation_changelog_previous_years
+description: Explore major updates for Azion's products and services, including Edge
+  Applications, Data Stream, Edge Firewall adjustments and API V3 developments.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/changelog/previous-years/previous-years-changelog.mdx:4]:
```suggestion
  Application, Data Stream, Edge Firewall adjustments, and API V3 developments.
```

File: src/content/docs/en/pages/deploy-journey/configure-node/authorize-node/authorize-node.mdx [hunk 18]
```diff
@@ -1,11 +1,13 @@
 ---
 title: How to authorize an edge node
-description: Authorize an edge node on the platform to start orchestrating services and resources at the edge.
-meta_tags: deploy, orchestrator
+description: Learn how to authorize and deploy Edge Nodes using Azion Console and
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/deploy-journey/configure-node/authorize-node/authorize-node.mdx:3]:
```suggestion
description: Learn how to authorize and deploy edge nodes using Azion Console and
```

File: src/content/docs/en/pages/devtools/api-graphql/queries-examples/queries-examples.mdx [hunk 19]
```diff
@@ -1,11 +1,13 @@
 ---
 title: Queries examples
-description: Find examples for Azion GraphQL queries in a dedicated repository.
-meta_tags: graphql, api, azion, query
-namespace: documentation_graphql_queries_examples
+description: Explore Azion's dedicated repository for GraphQL API query examples.
+  Find updated query samples for Data Stream, Edge Applications, and more on GitHub.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/api-graphql/queries-examples/queries-examples.mdx:4]:
```suggestion
description: Explore Azion's dedicated repository for GraphQL API query examples.
  Find updated query samples for Data Stream, Edge Application, and more on GitHub.
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/javascript-examples/firewall-add-response-header/firewall-add-response-header.mdx [hunk 20]
```diff
@@ -1,10 +1,14 @@
 ---
 title: JavaScript Examples - Adding a response header
-description: Based on the country code, accessed through event.request.metadata['geoip_country_code'], a response header is added through event.addResponseHeader().
-meta_tags: edge computing, javascript, edge functions
-namespace: documentation_products_edge_functions_javascript_examples_adding_response_header
+description: Explore how geo-location data in Edge Computing enables dynamic web application
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/javascript-examples/firewall-add-response-header/firewall-add-response-header.mdx:3]:
```suggestion
description: Explore how geo-location data in edge computing enables dynamic web application
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/javascript-examples/firewall-general-example/firewall-general-example.mdx [hunk 21]
```diff
@@ -1,10 +1,13 @@
 ---
 title: JavaScript Examples - General edge functions on firewall example
-description: General example for edge functions to run on Edge Firewall.
-meta_tags: edge computing, javascript, edge functions
-namespace: documentation_products_edge_functions_javascript_examples_general_firewall_example
+description: Explore JavaScript-based Edge Functions for robust firewall interactions,
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/javascript-examples/firewall-general-example/firewall-general-example.mdx:3]:
```suggestion
description: Explore JavaScript-based edge functions for robust firewall interactions,
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/runtime-apis/functions-objects/2023-04-07-index.mdx [hunk 22]
```diff
@@ -1,10 +1,15 @@
 ---
 title: Azion Runtime supported types
-description: Supported types for Azion Runtime.
-meta_tags: Supported types
-namespace: documentation_products_edge_functions_runtime_apis_javascript_supported_types
+description: Explore supported JavaScript data types and objects for Azion Edge Runtime,
+  including WebAssembly, Crypto API, and event handling, enhancing Edge computing
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/runtime-apis/functions-objects/2023-04-07-index.mdx:4]:
```suggestion
  including WebAssembly, Crypto API, and event handling, enhancing edge computing
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/azion-overview/azion-overview.mdx [hunk 23]
```diff
@@ -1,10 +1,13 @@
 ---
 title: Azion CLI
-description: Dive into Azion CLI, your tool for optimized edge management. Learn commands, features, and that allow you to interact with Azion products via terminal.
-meta_tags: cli, next, javascript, astro, vue, react, edge computing
+description: 'Explore Azion CLI for edge computing: manage and deploy edge applications,
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli/azion-overview/azion-overview.mdx:3]: It has '' in the description, different from others.

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/delete/delete.mdx [hunk 24]
```diff
@@ -1,10 +1,13 @@
 ---
 title: Azion CLI delete
-description: Delete your edge applications, edge functions and other resources through Azion CLI.
-meta_tags: cli, edge, delete, development
+description: Learn to manage your Edge resources using Azion CLI, including deleting
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/delete/delete.mdx:3]:
```suggestion
description: Learn to manage your edge resources using Azion CLI, including deleting
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/azion-overview/azion-overview.mdx (hunk 23 above)
Comment (gabriel-azion) [reply src/content/docs/en/pages/devtools/cli/azion-cli/azion-overview/azion-overview.mdx:3]:
```suggestion
description: Explore Azion CLI for edge computing: manage and deploy edge applications,
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/azion-overview/azion-overview.mdx [hunk 25]
```diff
@@ -1,10 +1,13 @@
...
+  integrate with Azion APIs, and optimize with Go SDK.'
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/devtools/cli/azion-cli/azion-overview/azion-overview.mdx:4]:
```suggestion
  integrate with Azion APIs, and optimize with Go SDK.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/dev-command/dev.mdx [hunk 26]
```diff
@@ -1,10 +1,13 @@
 ---
 title: Azion CLI dev
-description: Initialize a local development server.
-meta_tags: cli, next, javascript, astro, vue, react, edge computing
+description: Discover how to build and run Azion Edge Applications locally using the
+  dev command. Learn to link projects and use Edge Functions effectively.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/dev-command/dev.mdx:4]:
```suggestion
  dev command. Learn to link projects and use edge functions effectively.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/init-build-deploy/azion-create-deploy-application.mdx [hunk 27]
```diff
@@ -1,10 +1,13 @@
 ---
 title: Azion CLI init, build, and deploy commands
-description: Create and deploy a new application from Azion's templates.
-meta_tags: cli, next, javascript, astro, vue, react, edge computing
+description: Learn to initialize, build, and deploy applications using Azion's Edge
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/init-build-deploy/azion-create-deploy-application.mdx:3]:
```suggestion
description: Learn to initialize, build, and deploy applications using Azion Edge
```

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/link-project/link.mdx [hunk 28]
```diff
@@ -1,10 +1,13 @@
 ---
 title: Azion CLI link
-description: Linking an existing project to run on the Azion Edge platform.
-meta_tags: cli, next, javascript, astro, vue, react, edge computing
+description: Learn how to link and deploy projects on the Azion Edge platform, using
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/link-project/link.mdx:3]:
```suggestion
description: Learn how to link and deploy projects on the Azion Edge Platform, using
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/devtools/cli/azion-cli/azion-overview/azion-overview.mdx [hunk 29]
```diff
@@ -1,10 +1,13 @@
 ---
 title: Azion CLI
-description: Dive into Azion CLI, your tool for optimized edge management. Learn commands, features, and that allow you to interact with Azion products via terminal.
-meta_tags: cli, next, javascript, astro, vue, react, edge computing
+description: Explore Azion CLI for edge computing: manage and deploy edge applications,
+  integrate with Azion APIs, and optimize with Go SDK.
+permalink: /documentation/products/azion-cli/overview/
+meta_tags: Azion CLI, edge computing, edge applications, Jamstack, Go SDK, web frameworks,
+  SSR, deployment, API integration, Azion products
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli/azion-overview/azion-overview.mdx:7]:
```suggestion
description: Explore Azion CLI for edge computing: manage and deploy edge applications, integrate with Azion APIs, and optimize with Go SDK.
permalink: /documentation/products/azion-cli/overview/
meta_tags: Azion CLI, edge computing, edge applications, Jamstack, Go SDK, web frameworks, SSR, deployment, API integration, Azion products
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/javascript-examples/respond-site/respond-site.mdx [hunk 30]
```diff
@@ -1,10 +1,13 @@
 ---
 title: JavaScript Examples - Respond with Another Site
-description: Respond to the request with a response from another website.
-meta_tags: edge computing, javascript, edge functions, respond with another site
-namespace: documentation_products_edge_functions_javascript_examples_respond_another_site
+description: Explore how to implement temporary redirects using JavaScript with the
+  fetch API on the Azion platform, enhancing web development and response strategies.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/javascript-examples/respond-site/respond-site.mdx:4]:
```suggestion
description: Explore how to implement temporary redirects using JavaScript with the fetch API on the Azion platform, enhancing web development and response strategies.
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/javascript-examples/respond-site/respond-site.mdx [hunk 31]
```diff
@@ -1,10 +1,13 @@
...
 permalink: documentation/products/edge-application/edge-functions/javascript-examples/respond-site
+meta_tags: edge computing, javascript, edge functions, temporary redirects, fetch
+  API, web development, site redirection, Azion platform, technology solutions
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/javascript-examples/respond-site/respond-site.mdx:7]:
```suggestion
meta_tags: edge computing, javascript, edge functions, temporary redirects, fetch API, web development, site redirection, Azion platform, technology solutions
```

**Pull request merged**